docs/content/en/docs-dev/operator-manual/piped/configuration-reference.md

```diff
@@ -222,8 +222,8 @@ Must be one of the following structs:
 | ignoreGroups | []string | List of event groups should be ignored. | No |
 | apps | []string | List of applications where their events should be routed to the receiver. | No |
 | ignoreApps | []string | List of applications where their events should be ignored. | No |
-| envs | []string | List of environments where their events should be routed to the receiver. | No |
-| ignoreEnvs | []string | List of environments where their events should be ignored. | No |
+| labels | map[string]string | List of labels where their events should be routed to the receiver. | No |
+| ignoreLabels | map[string]string | List of labels where their events should be ignored. | No |
```

Comment:
Is env removed completely in next release? If not, I think we should add deprecated instead of deleting from docs.

Reply:
for the current implementation, since I removed the env related code completely from the piped source, env config in piped config will not work any more. I only keep that field on config package in order to not break the running piped with it old configuration. As we discussed on `Remove EnvironmentStore` PR, I think instead of making env deprecated on piped side, should remove it on the next release, so that we can remove other parts (on servers) which related to env on the next next release, wdyt?

Reply:
Oh, I see. That's sound good.
TYVM :D